src/js/common/containers/BetaApp.jsx

```diff
@@ -2,59 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import objectValues from 'lodash/fp/values';
-import { registerBeta, isUserRegisteredForBeta } from '../actions';
-import AlertBox from '../components/AlertBox';
-
-export const features = {
-  healthAccount: 'health_account',
-  veteranIdCard: 'veteran_id_card',
-  profile: 'profile',
-  dashboard: 'dashboard'
-};
+import { isUserRegisteredForBeta } from '../../beta-enrollment/actions';
```

Comment:
This file is shortened dramatically, because now it ultimately just confirms that they are enrolled in the corresponding beta, and redirects them if not. The logic to register them in the beta has been moved into the beta-enrollment project.